docs/register/parameters.rst

```diff
@@ -95,4 +95,4 @@ When :doc:`resolving with parameters <../resolve/parameters>`, your lambda will
       SomeValue = "../"
     };
 
-    var reader = scope.Resolve<MyConfig>(new NamedParameter(config, customConfig));
\ No newline at end of file
+    var worker = scope.Resolve<Worker>(new NamedParameter(config, customConfig));
```

Comment:
```suggestion
    var worker = scope.Resolve<Worker>(new NamedParameter("config", customConfig));
```

Reply:
`"config"` on line 87 also should be in quotes.

Reply:
good catch, changed.

Reply:
...also on line 87. I can't make a comment on that line or I'd do it. Thanks!

Reply:
sorry I missed the one on line 87. done, thanks.